docs/framework.md

```diff
@@ -173,16 +162,7 @@ P   U   U   U   U   U
 
 </div>
 
- The `publicAuthority` section of an OCDS release should be used to reference the entry for the sponsoring agency or department in the `parties` section. 
-
-**Schema**: Information can be provided using the following OCDS fields.
-
-```eval_rst
-.. jsonschema:: _static/patched/release-schema.json
-    :include: publicAuthority
-    :collapse: 
-    :nocrossref:
-```
+A reference to the sponsoring agency or department should be provided using an [OrganizationReference](https://standard.open-contracting.org/1.1/en/schema/reference/#organizationreference) object in the [`publicAuthority`](../reference/schema/#release-schema.json,,publicAuthority) field, referencing the relevant entry in the `parties` section.
```

Comment:
It would be better for the link to the parent field to be before the link to the object reference, since the former includes the content from the latter, so the reader need only click one link instead of two to get the information that was originally included in the schema tables.

Reply:
Once a user learns what an OrganizationReference is, they don't need to follow that link anymore. This construction is used everywhere for that reason: provide the more fundamental information first.

If we change it here we should change it everywhere, but I think the sentence flows better as-is, and it introduces more fundamental concepts first. Users only need to click twice the first time they see an OrgRef link.